tests/Behat/CorporateBlocksContext.php

```diff
@@ -51,19 +51,6 @@ public function gatherContexts(BeforeScenarioScope $scope) {
     $this->configContext = $environment->getContext(ConfigContext::class);
   }
 
-  /**
-   * Backup site configuration.
-   *
-   * @param \Behat\Behat\Hook\Scope\BeforeScenarioScope $scope
-   *   The scenario scope.
-   *
-   * @beforeScenario @preserve-site-switcher-configuration
-   */
-  public function backupSiteSwitcherConfiguration(BeforeScenarioScope $scope): void {
-    $value = \Drupal::config('oe_corporate_blocks.data.site_switcher')->get('active');
-    $this->configContext->setConfig('oe_corporate_blocks.data.site_switcher', 'active', $value);
-  }
-
```

Comment:
Probably this method should contain some exception regarding legacy functionality, or should be removed...
`assertNotActiveSiteSwitcherLink`
`assertActiveSiteSwitcherLink`

Reply:
I forgot a lot of code connected to site switcher.

Reply:
Good catch